packages/debugger/src/tokens.ts

```diff
@@ -490,12 +535,47 @@ export namespace IDebugger {
       };
     }
 
+    /**
+     * Arguments for 'inspectVariables' requests.
+     */
+    export interface IInspectVariablesArguments {}
+
     export interface IInspectVariablesResponse extends DebugProtocol.Response {
       body: {
         variables: DebugProtocol.Variable[];
       };
     }
 
+    /**
+     * Arguments for 'richVariables' request
+     *
+     * This is an addition to the Debug Adapter Protocol to support
+     * render rich variable representation.
+     */
+    export interface IRichVariablesArguments {
+      /**
+       * Variable name
+       */
+      variableName: string;
+      /**
+       * Frame Id
+       */
+      frameId?: number;
+    }
+
+    /**
+     * Arguments for 'richVariables' request
+     *
+     * This is an addition to the Debug Adapter Protocol to support
+     * render rich variable representation.
```

Comment:
Could be phrased like this maybe?

```suggestion
     * rich rendering of variables.
```